tests/test-runner/include/logapi.shlib

```diff
@@ -199,11 +199,11 @@ function log_neg_expect
 		print -u2 $($out)
 		_printerror "$@" "unexpectedly exited $status (File not found)"
 	# bus error - core dump
```

Comment:
nit: let's make it explicit where 263 (and 267) come from in the comments.  Perhaps something like this.

```
# bus error - core dump (256+signal, SIGBUS=7)
```

Reply:
Will do.